pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -3265,3 +3272,361 @@ def get_fold_generator(
                     start_with_window=True,
                 )
         return fold_generator
+
+    def test_model(
+        self,
+        estimator: Optional[Any] = None,
+        test: str = "all",
+        alpha: float = 0.05,
+        split: str = "all",
+    ) -> pd.DataFrame:
+        if estimator is None:
+            data = self._get_y_data(split=split)
+            results = test_(data=data, test=test, alpha=alpha)
+        else:
+            raise NotImplementedError(
+                "Tests on estimators have not been implemented yet."
+            )
+        return results
+
+    def _get_y_data(self, split="all"):
+        if split == "all":
+            data = self.y.copy()
+        elif split == "train":
+            data = self.y_train.copy()
+        else:
+            raise ValueError("split value: '{split}' is not supported.")
+        return data
+
+
+#############################################################
+###################### TESTS STARTS HERE ####################
+#############################################################
```

Comment:
I think we should move this into a separate file

Reply:
OK, will do.

Reply:
Any preference of where this should go?

Reply:
You can make a time_series_tests.py file in internal

Reply:
done, but added a separate module since we can have more than time series tests.